package.json

```diff
@@ -27,6 +29,7 @@
     "@babel/plugin-transform-runtime": "^7.3.4",
     "@babel/preset-env": "^7.3.4",
     "@babel/preset-react": "^7.0.0",
+    "apollo-boost": "^0.3.1",
```

Comment:
I think `apollo-boost`  is not used any more because we directly use `apollo-client`.
But anyways we can remove it in the future PR.